graphql-ballerina/tests/utils.bal

```diff
@@ -22,9 +22,11 @@ isolated function getJsonPayloadFromService(string url, string document, string?
 returns json|error {
     http:Client httpClient = check new(url);
     if (operationName is string) {
-        return check httpClient->post("/", { query: document, operationName: operationName }, targetType = json);
+        http:Response response = check httpClient->post("/", { query: document, operationName: operationName });
+        return response.getJsonPayload();
     }
-    return check httpClient->post("/", { query: document }, targetType = json);
+    http:Response response = check httpClient->post("/", { query: document });
+    return response.getJsonPayload();
```

Comment:
What's the reason for this change? We can even simply this as follows:

```suggestion
        http:Client httpClient = check new(url);
        return httpClient->post("/", { query: document, operationName: operationName });
```

Reply:
If we return ```httpClient->post("/", { query: document, operationName: operationName })``` here, tests returning responses with errors will hit the ```check``` at ```json actualPayload = check getJsonPayloadFromService(url, document)```.